test/load/allocation/grpc/README.md

```diff
@@ -74,3 +74,186 @@ You can use environment variables overwrite defaults. To run only a single run o
 ```
 TESTRUNSCOUNT=1 ./runAllocation.sh 40 10
 ```
+
+
+# Running Scenario tests
+
+The scenario test allows you to generate a variable number of allocations to
+your cluster over time, simulating a game where clients arrive in an unsteady
+pattern.
```

Comment:
```suggestion
The scenario test allows you to generate a variable number of allocations to
your cluster over time, simulating a game where clients arrive in an unsteady
pattern. It is also setup such that game servers will shut down after a set time period,
also simulating the GameServer churn that is also expected during normal game play.
```

Although I think GameServers go back to `Ready` state, yeah? I think we should be clear about how that works.

Reply:
I added a second sentence here and also a bit more in the Fleet Setting section below to highlight the two flags that we set on the simple game server.